### ISSUE TITLE
Missing discount data in the subscription model

### ISSUE THREAD

**Issue by archatas** [websideproject/paddle-billing-client#165]
## 🐛 Bug Report

Subscriptions miss the data about the discount used.
Here are the official Paddle Billing docs on subscriptions:
https://developer.paddle.com/api-reference/subscriptions/overview

## 🔬 How To Reproduce

1. Create a subscription at Paddle Billing.
2. Call `client.list_subscriptions()`

### Environment

* OS: macOS
* Python 3.10

**Comment by bgervan** [comment]
All the models allow the extra data

```
class LazyBaseModel(BaseModel):
    class Config:
        extra = "allow"
```

If something is not defined explicitly in the model, you can still use subscription.discount if the API returned that.

But I will update the code, take note, that the subscription response is not a real discount object, just a few property exist, and this is true for the subscription request too

**Comment by bgervan** [comment]
Released with v0.1.14

**Comment by archatas** [comment]
OK. Thank you. Good to know.
